source/upgrade/important-upgrade-notes.rst

```diff
@@ -5,15 +5,88 @@ Important Upgrade Notes
+|                                                    |                                                                                                                                                                  |
+|                                                    | **Table locks or impact to existing operations on the table:**                                                                                                   |
+|                                                    |                                                                                                                                                                  |
+|                                                    | Table locks - Threads table                                                                                                                                      |
+|                                                    |                                                                                                                                                                  |
+|                                                    | Zero downtime is possible when upgrading to this release.                                                                                                        |
+|                                                    |                                                                                                                                                                  |
+|                                                    | Queries posted above can be run prior to upgrading Mattermost for both MySQL and PostgreSQL. After schema changes migration becomes instantaneous (0.78 sec).    |
+|                                                    +------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|                                                    | Starting with the Calls version shipping with v7.7, there's now a minimum version requirement when using the external RTCD service. This means that if Calls is  |
+|                                                    | configured to use the external service, customers need to upgrade RTCD first to at least version 0.8.0 or the plugin will fail to start.                         |
+|                                                    +------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|                                                    | Boards will be served as an in-built product from within Mattermost server instead of a plugin. If you want to disable this, please set the                      |                     
```

Comment:
I realize the last sentence can be confusing if thought to be connected to the previous sentence. Let's change that to "While running in product mode, the boards plugin will remain disabled"